### PR TITLE
Fixed titles grammar and "Camelish" caps

### PR DESCRIPTION
**Pull request by ZETA-15**
I have fixed some grammar in the titles that sounded awkward and solved the caps on every word that is simply disgusting.

Regards

Alberto

### CODE REVIEW

File: tutorials/linux-setup-gre-tunnel/01.en.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ curl http://www.cpanel.net/showip.cgi --interface 10.0.0.2
 
 At the end you should see Server A's IP.
 
-## Step 5 - Forward Ports
+## Step 5 - Ports forwarding
```
Comment (n-se) [inline tutorials/linux-setup-gre-tunnel/01.en.md]:
```suggestion
## Step 5 - Port forwarding
```

**Comment by leojakeson** [comment]
Thanks for the suggestions Alberto.

Right now all of the step descriptions in this tutorial start with a verb. The suggested changes would remove that, but only for half of the steps, disrupting the pattern.

My recommendation would be to either keep the verb at the beginning, or to change all of the steps. Removing the capitalization for the additional words is not a problem.

Option A: Load module; Setup tunnel; Add new routes; Configure NAT; Forward ports.

Option B: Module loading; Tunnel setup; New routes; NAT configuration; Port forwarding.

Since this is your tutorial @Fabrimat please let us know which option you want, so we can implement that.

**Comment by Fabrimat** [comment]
> Thanks for the suggestions Alberto.
> 
> Right now all of the step descriptions in this tutorial start with a verb. The suggested changes would remove that, but only for half of the steps, disrupting the pattern.
> 
> My recommendation would be to either keep the verb at the beginning, or to change all of the steps. Removing the capitalization for the additional words is not a problem.
> 
> Option A: Load module; Setup tunnel; Add new routes; Configure NAT; Forward ports.
> 
> Option B: Module loading; Tunnel setup; New routes; NAT configuration; Port forwarding.
> 
> Since this is your tutorial @Fabrimat please let us know which option you want, so we can implement that.

I think that option B is the best one

**Comment by ZETA-15** [comment]
Dear Fabrizio

It would be probably better to write "New routes addition" or "New routes
Implementation" to give more context to the chapter.

Best

Alberto

Il mer 26 feb 2020, 14:24 Fabrizio La Rosa <notifications@github.com> ha
scritto:

**Pull request merged**